packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue

```diff
@@ -1,28 +1,69 @@
 <template>
-    <div
-        :class="$style['c-takeawaypayActivation']"
-        data-test-id="takeawaypayActivation">
-        {{ copy.text }}
+    <div :class="$style['c-takeawaypayActivation']">
+        <card-component
+            is-rounded
+            is-page-content-wrapper
+            card-heading-position="center"
+            data-test-id="takeawaypay-activation-component"
+            :class="$style['c-takeawaypayActivation-card']">
+            <activation-logged-in
+                :login-url="loginUrl"
+                :registration-url="registrationUrl" />
+            <activation-not-logged-in
+                :login-url="loginUrl"
+                :registration-url="registrationUrl" />
+            <activation-successful
+                :home-url="homeUrl" />
+            <activation-failed />
```

Comment:
I guess the `v-if` attributes to choose which one of these components to use will be added in a later PR?

Reply:
Yep. Will be in the separate ticket when we will introduce the actual business logic